README.md

```diff
@@ -1,9 +1,7 @@
-Library of bpipe functions for processing minc files
+# Library of bpipe functions for processing minc files
 ====================================================
 
-minc-bpipe-library provides a set of chainable minc file processing functions to (pre)process data.
-
-It requires http://www.bic.mni.mcgill.ca/ServicesSoftware/ServicesSoftwareMincToolKit, and https://github.com/ssadedin/bpipe/
```

Comment:
Can you actually leave this in please, and add that it needs gnu-parallel.

I'd like (in theory) that others use it.